CONTRIBUTING.md

```diff
@@ -103,7 +103,7 @@ Similarly, a Deprecation section should start with "DEPRECATED: " followed by a
 ### Check commit message locally:
 
 ```
-$ npm install commitlint@latest @commitlint/config-angular@latest
+$ npm install commitlint@latest @commitlint/config-angular@latest @commitlint/config-conventional@latest
```

Comment:
IIRC, the code originally used `@commitlint/config-angular@latest` and then it was changed to `@commitlint/config-conventional@latest`, probably forgetting about the docs.

It would make sense to check if `@commitlint/config-angular@latest` is still needed (I think it isn't) and if not, remove it from the instructions.

Reply:
Just gave it a spin. I can confirm that `@commitlint/config-angular@latest` isn't needed anymore.
@ConorC117 can you remove the module from the install command?

Reply:
Thanks for clarifying that @pvoborni and for checking @FKolwa.
Just removed the module from the install command. If anything else is needed, just let me know!